### DISCUSSION TITLE
Bundle size 156kb

### DISCUSSION THREAD

**Discussion by scofield-ua** [livewire/livewire#4491] [1. Help]
Hey guys. Isn't such a size is a bit too much? And it's just a livewire, without including any other libraries.

Are you planning to work on reducing the bundle size? What are your thoughts, maybe I'm missing something?

**Comment by joshhanley** [comment]
@scofield-ua hmm not sure where you got that number from, as the Livewire bundle size is ~42kb

![image](https://user-images.githubusercontent.com/882837/148719417-88ca33bd-d129-41f0-911c-e5954e6a5370.png)

That screenshot is from loading a Livewire app with browser asset caching turned off, so you can see it's pretty small 🙂

Hope this helps!

**Reply by scofield-ua** [reply]
@joshhanley  this is what I got:
![image](https://user-images.githubusercontent.com/1827843/148736975-37624429-bf7e-4c1d-8aa3-d7b726afdaf8.png)

That's actually same size from the repository: https://github.com/livewire/livewire/blob/master/dist/livewire.js

**Reply by joshhanley** [reply]
@scofield-ua ah interesting, I've tested it on a few different Livewire projects now and I still get the same ~42kb back each time.

If you go to my website https://joshhanley.com.au/ what size do you get for LW's bundle?

Also what setup are you running locally mac/windows/linux and valet/homestead/etc?

**Reply by scofield-ua** [reply]
@joshhanley like ~40kb. That's probably because of gzip compression, so I guess it's fine now.

**Reply by joshhanley** [reply]
@scofield-ua ah yep of course! Makes sense. Ok sweet 🙂
